lambdas/reset-identity/src/test/java/uk/gov/di/ipv/core/resetidentity/ResetIdentityHandlerTest.java

```diff
@@ -83,18 +80,61 @@ void setUpEach() {
     }
 
     @Test
-    void shouldDeleteUsersVcsAndReturnNext() throws SqsException {
+    void handleRequest_whenFraudReset_shouldDeleteUsersVcsAndReturnNext() throws SqsException {
+        // Arrange
+        when(ipvSessionService.getIpvSession(TEST_SESSION_ID)).thenReturn(ipvSessionItem);
+        when(clientOAuthSessionDetailsService.getClientOAuthSession(any()))
+                .thenReturn(clientOAuthSessionItem);
+        ProcessRequest event =
+                ProcessRequest.processRequestBuilder()
+                        .ipvSessionId(TEST_SESSION_ID)
+                        .ipAddress(TEST_CLIENT_SOURCE_IP)
+                        .clientOAuthSessionId(TEST_CLIENT_SOURCE_IP)
+                        .journey(TEST_JOURNEY)
+                        .featureSet(TEST_FEATURE_SET)
+                        .lambdaInput(Map.of(IS_USER_INITIATED, false))
+                        .build();
+
+        // Act
+        JourneyResponse journeyResponse =
+                objectMapper.convertValue(
+                        resetIdentityHandler.handleRequest(event, context), JourneyResponse.class);
+
+        // Assert
+        verify(verifiableCredentialService).deleteVcStoreItems(TEST_USER_ID, false);
+        verify(criResponseService).deleteCriResponseItem(TEST_USER_ID, F2F_CRI);
+        verifyNoInteractions(mockAuditService);
+        verifyNoInteractions(emailService);
+        assertEquals(JOURNEY_NEXT.getJourney(), journeyResponse.getJourney());
+    }
+
+    @Test
+    void handleRequest_whenUserInitiated_shouldSendEmailAndRaiseAuditLog() throws SqsException {
+        // Arrange
         when(ipvSessionService.getIpvSession(TEST_SESSION_ID)).thenReturn(ipvSessionItem);
         when(clientOAuthSessionDetailsService.getClientOAuthSession(any()))
                 .thenReturn(clientOAuthSessionItem);
+        ProcessRequest event =
+                ProcessRequest.processRequestBuilder()
+                        .ipvSessionId(TEST_SESSION_ID)
+                        .ipAddress(TEST_CLIENT_SOURCE_IP)
+                        .clientOAuthSessionId(TEST_CLIENT_SOURCE_IP)
+                        .journey(TEST_JOURNEY)
+                        .featureSet(TEST_FEATURE_SET)
+                        .lambdaInput(Map.of(IS_USER_INITIATED, true))
+                        .build();
+        when(emailServiceFactory.getEmailService()).thenReturn(emailService);
 
+        // Act
         JourneyResponse journeyResponse =
                 objectMapper.convertValue(
                         resetIdentityHandler.handleRequest(event, context), JourneyResponse.class);
 
+        // Assert
         verify(verifiableCredentialService).deleteVcStoreItems(TEST_USER_ID, true);
         verify(criResponseService).deleteCriResponseItem(TEST_USER_ID, F2F_CRI);
         verify(mockAuditService, times(1)).sendAuditEvent((AuditEvent) any());
+        verify(emailService, times(1)).sendUserTriggeredIdentityResetConfirmation(any(), any());
```

Comment:
Would it be worth asserting the actual email and name here?

Reply:
The email address is simple, but the name would require quite a lot of mocking out, so maybe not worth it.